### PR TITLE
Correctly display artifact indexes.

### PR DESCRIPTION
**Pull request by mikalstill**
We were displaying the maximum index number, but not the number of
indexes actually present, which made me think the version cleanup
code wasn't working.

### CODE REVIEW

File: shakenfist_client/commandline/artifact.py [hunk 1]
```diff
@@ -69,17 +69,21 @@ def artifact_list(ctx, node=None):
         x = PrettyTable()
         x.field_names = ['uuid', 'type', 'source url', 'versions', 'state']
         for meta in artifacts:
+            versions = '%d of %d' % (len(meta.get('blobs', [])),
+                                     meta.get('index', 'unknown'))
```
[the remark this answers is not in the document]
Comment (mikalstill) [reply shakenfist_client/commandline/artifact.py:73]: So, the previous version had "index". I guess the question is what are we trying to say here. I read the entry as "there are three versions, of a historic 75", wherewas you obviously read it as "there are three versions, which is your quota for this artifact". I'm not sure which is right.

[the remark this answers is not in the document]
Comment (mandoonandy) [reply shakenfist_client/commandline/artifact.py:73]: I see your reading. I can see the curiosity value of how many total versions ever existed. I also see value in how many the system will store before deletions occur.

Maybe drop the second value. It makes it hard to parse. Just display actual number of versions stored?

The user can request all three values with a show command. ("Full details available upon request.")
